docusaurus.config.js

```diff
@@ -0,0 +1,111 @@
+// @ts-check
+
+const GITHUB_ORG = "comcode-org";
+const GITHUB_PROJECT = "hackmud_wiki";
+
+// See: https://docusaurus.io/docs/api/docusaurus-config
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  /**
+   * GENERAL CONFIG
+   */
+  // Metadata
+  title: "hackmud Wiki",
+  tagline: "Knowledge repository for the video game hackmud",
+  //favicon: 'img/favicon.ico',
+
+  // Deployment details
+  url: `https://${GITHUB_ORG}.github.io`,
+  baseUrl: `/${GITHUB_PROJECT}/`,
+
+  // GitHub Pages config for CLI deployment
+  organizationName: GITHUB_ORG,
+  projectName: GITHUB_PROJECT,
+
+  // Fail loudly on broken links
```

Comment:
'fail loudly' does this mean break the build? (if so, then great; but also let's be more clear in the comment)

Reply:
It emits an error and the build fails, yeah. I went to double-check usage though and it turns out the following two lines are already the default values, so I just removed them in https://github.com/comcode-org/hackmud_wiki/pull/2/commits/0085879e3fc4a073af024a95febfd5dfe1569ab3.

Reply:
Love it. Thanks!